app/pages/analyst/application/[applicationId]/rfi/[rfiId]/index.tsx

```diff
@@ -9,6 +9,9 @@ import { RfiIdQuery } from '__generated__/RfiIdQuery.graphql';
 
 const getRfiIdQuery = graphql`
   query RfiIdQuery($rowId: Int!) {
+    rfiDataByRowId(rowId: $rowId) {
```

Comment:
This `rowId` is using the `applicationId` as you can see in the `variablesFromContext` at the bottom of the file so it returns the incorrect rfi to edit.

Easy to see how this would be confusing given the poorly named query due to relay naming conventions. Though this query still needs the `applicationId` as it is used in the `AnalystLayout_query` fragment.

Reply:
Ah thanks, great catch